source/blog/_posts/2018-01-12-mutation-testing-with-infection.md

```diff
@@ -0,0 +1,374 @@
+---
+title: Mutation Testing with Infection
+tags:
+- phpunit
+- infection
+- testing
+
+categories:
+- dev
+
+authors: [tn]
+github_link: blog/_posts/2018-01-12-mutation-testing-with-infection.md
+
+---
+
+<div class="alert alert-info">
+Anyone who is just searching for the updated example source code and does not want to read the complete blog post, <a href="https://github.com/teiling88/mutation-testing">here it is.</a>
+</div>
+
+This blog post is a follow up for my mutation testing post from [2017-08-24](https://developers.shopware.com/blog/2017/08/24/mutation-testing/). Since [31.12.2017](https://github.com/humbug/humbug/commit/53730b3306efebf85bd66b6f7ec870d500f5ccbd) humbug is marked as deprecated with an link to infection as an alternative. Maks Rafalko is the author of infection and well known as borNfreee on github. The last released version of Infection is 0.7.0. So let us have a closer look into it.  
```

Comment:
... is marked as deprecated with **a** link

Reply:
fixed